app/views/projects/reports.html.erb

```diff
@@ -4,82 +4,143 @@
-    <% end %>
-  <% end %>
-  <% if params[:since].present? %>
-    <%= link_to t('projects.reports.all_time'), reports_project_path(@project), class: "btn btn-warning btn-sm navbar-btn navbar-btn-reports" %>
-  <% end %>
 <% end %>
 
 <% content_for :navbar do %>
   <%= render 'projects/navbar', project: @project %>
 <% end %>
 
 <div class="row">
-  <div class="col-sm-12 col-md-12">
-    <% calculate_and_render_burn_up! %>
-
-    <h3><%= t('projects.reports.burn_up') %></h3>
-    <p>The current backlog has a total of <strong><%= @total_backlog_points %> points</strong>. This is without counting the stories in the Chilly Bean!</p>
-    <p>At the current velocity (<%= @service.velocity %> points) the existing backlog should only be finished around the iteration <%= last_iteration_number %> (<%= last_iteration_start_date %>).</p>
-
-    <% if last_iteration_number == last_iteration_number(true) %>
-      <p>If the velocity keeps up this pace we should be on track to the date above.</p>
-    <% else %>
-      <p>But with a standard deviation of <%= "%3.2f" % Math.sqrt(@service.standard_deviation) %>, it's more realistic to expect the project to finish around <strong>iteration <%= last_iteration_number(true) %></strong> (<%= last_iteration_start_date(true) %>), if the velocity doesn't increase or the backlog is not simplified.</p>
-    <% end %>
-
-    <p><%= area_chart @group_by_day, max: @total_backlog_points, xtitle: t('day'), ytitle: t('accepted_points') %></p>
+  <div class="col-xs-12">
+    <div class="page-header">
+      <h4 class="page-header-title darkgrey-2">
+        <%= t('reports') %>
+      </h4>
+    </div>
+  </div>
+</div>
 
+<div class="row">
+  <div class="col-xs-12">
+    <div class="panel panel-default card">
+      <div class="panel-body">
+        <ul class="nav nav-justified nav-pills reports-list col-xs-12" id="reports-list">
+          <li class="reports-list-item">
+            <a href="#burn-up" class="nav-link"> <%= t('projects.reports.burn_up') %> </a>
+          </li>
+          <li class="reports-list-item">
+            <a href="#current-iteration" class="nav-link"> <%= t('projects.reports.current_iteration') %></a>
+          </li>
+          <li class="reports-list-item">
+            <a href="#velocity-by-iteration" class="nav-link"> <%= t('projects.reports.velocity_by_iteration') %></a>
+          </li>
+          <li class="reports-list-item">
+            <a href="#velocity-by-member" class="nav-link"> <%= t('projects.reports.velocity_by_member') %></a>
+          </li>
+          <li class="reports-list-item">
+            <a href="#bugs-impact" class="nav-link"> <%= t('projects.reports.bugs_impact') %></a>
+          </li>
+        </ul>
+      </div>
+
+      <div class="panel-footer clearfix reports-filters">
+        <div class="reports-page-actions">
+          <% unless params[:since].nil? %>
```

Comment:
You could use just `if params[:since]` instead of checking the inverse (with `unless`) of a falsy (`nil`) value.